modules/porous_flow/doc/content/source/bcs/PorousFlowSinkBC.md

```diff
@@ -0,0 +1,13 @@
+# PorousFlowSinkBC
+
+Proxy class that allows users to use the BC syntax for specifying enthalphy sink.
```

Comment:
Perhaps this should read " Proxy class that allows users to use the BC syntax for specifying an enthalphy sink.  This class adds a (PorousFlowSink)[PorousFlowSink.md] and a (PorousFlowEnthalpySink)[PorousFlowEnthalpySink.md] "

Reply:
Sounds goods :+1: